controllers/status.go

```diff
@@ -71,6 +72,18 @@ func setVRGDataReplicatingCondition(conditions *[]metav1.Condition, observedGene
 	})
 }
 
+// sets conditions when Primary VRG data replication is established
+func setVRGDataReplicationEstablishedCondition(conditions *[]metav1.Condition, observedGeneration int64,
```

Comment:
By itself, the term `ReplicationEstablished` is great but we don't have that information from primary about it, which is why we're changing it to a bland 'Ready', right?  If yes, then I wonder why not simply use the `Ready` in the function name?  A function name of `setVRGDataReadyCondition` (instead of `setVRGDataReplicationEstablishedCondition`) will be consistent with the 'Ready` reason and is also in line with the method `setVRGClusterDataReadyCondition`.

Reply:
`DataReady` will conflict with the condition `DataReady`, this is one of the reasons for `DataReady`. So we should disambiguate it differently, say `setVRGAsPrimaryReadyCondition` or a variant thereof.

Reply:
ACK.

Will rename the function from "setVRGDataReplicationEstablishedCondition" to  "setVRGAsPrimaryReadyCondition"